### PR TITLE
[PD-1971] Add danger.js config

### PR DESCRIPTION
**Pull request by attilakiss323**
**JIRA**
https://capmo-team.atlassian.net/browse/PD-1971

### CODE REVIEW

**Review by kotielnikov — APPROVED** [review]
Cool! Thanks

File: Danger/dangerfile.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+function checkForVersionUpdate() {
+  const packageChanged = danger.git.modified_files.includes('package.json');
+  const lockfileChanged = danger.git.modified_files.includes(
+    'package-lock.json'
+  );
+
+  if (packageChanged) return;
+  if (packageChanged && !lockfileChanged) {
+    issues.push(
+      `Changes were made to package.json, but not to package-lock.json - <i>'Perhaps you need to run 'npm install'?'</i>`
```
Comment (across-the-grid) [inline Danger/dangerfile.js:117]:
```suggestion
      `Changes were made to \`package.json\`, but not to \`package-lock.json\` - Perhaps you need to run \`npm i\``
```

File: Danger/dangerfile.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    );
+  }
+
+  issues.push(`Please ensure the package version has been bumped in accordance with [Semantic Versioning](https://semver.org/)\n
+  and run 'npm i' to update package-lock\n`);
```
Comment (across-the-grid) [inline Danger/dangerfile.js:122]:
```suggestion
  issues.push(`Please ensure the package version has been bumped in accordance with [Semantic Versioning](https://semver.org/)\n
  and run \`npm i\` to update \`package-lock.json`\n`);
```

File: Danger/helpers.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    'needs_rebase',
+    'needs_testing',
+    'on_hold',
+    'prototype',
+    'refactoring',
+    'security',
+    'setup',
+    'wip',
+  ],
+};
```
Comment (across-the-grid) [inline Danger/helpers.js:66]:
```suggestion
const defaultLabels = [
 'bug',
    'dependencies',
    'enhancement',
    'needs_rebase',
    'needs_testing',
    'on_hold',
    'prototype',
    'refactoring',
    'security',
    'setup',
    'wip',
]
const labelsPerPlatform = {
  mobile: [
    ...defaultLabels,
    'release',
  ],
  web: [
     ...defaultLabels,
  ],
  backend: [
     ...defaultLabels,
    'infra',
  ],
};
```

Comment (across-the-grid) [reply Danger/helpers.js:66]: I would make this more DRY, because the list might be extended with Microservices (more repos) in the future.

**Review by across-the-grid — APPROVED** [review]

**Pull request merged**